htdocs/commande/class/commande.class.php

```diff
@@ -3714,6 +3714,48 @@ public function LibStatut($status, $billed, $mode, $donotshowbilled = 0)
 		return dolGetStatus($labelStatus, $labelStatusShort, '', $statusType, $mode, '', array('tooltip' => $labelTooltip));
 	}
 
+	/**
+	 * getTooltipContentArray
+	 * @param array $parameters
+	 * @since v18
+	 * @return array
+	 */
+	public function getTooltipContentArray($parameters)
+	{
+		global $conf, $langs, $user;
```

Comment:
Hello @frederic34 
thanks for your works.
it's possible to mutualized this function "getTooltipContentArray" in commonbject or not ?

Reply:
@hregis 
getTooltipContentArray is in every object, it returns an array with tooltîp data
getTooltipContent is in commonobject with hookmanager

Reply:
@frederic34 great ! thanks